src/main/java/crazypants/enderio/machine/spawner/TilePoweredSpawner.java

```diff
@@ -268,7 +268,7 @@ protected IPoweredTask createTask(IMachineRecipe nextRecipe, float chance) {
     } else {
       ticksDelay = TilePoweredSpawner.MAX_SPAWN_DELAY_BASE - ((TilePoweredSpawner.MAX_SPAWN_DELAY_BASE - TilePoweredSpawner.MIN_SPAWN_DELAY_BASE) / 2);
     }
-    ticksDelay >>= Math.min(3, getCapacitor().getTier() - 1);
+    ticksDelay >>= Math.min(3,getCapacitorType().ordinal());
```

Comment:
This is almost definitely wrong, the ordinal for e.g. the silver capacitor is 7, while its tier is 1, it should be equivalent to a basic capacitor.

Reply:
sorry for neglecting this